support/mender-inventory-geo

```diff
@@ -0,0 +1,68 @@
+#!/bin/sh
+#
+# Example script called by Mender agent to collect inventory data for a
+# particular devce. The script needs to be located in $(datadir)/mender and its
+# name shall start with `mender-inventory-` prefix. The script shall exit with
+# non-0 status on errors. In this case the agent will discard any output the
+# script may have produced.
+#
+# The script shall output inventory data in <key>=<value> format, one entry per
+# line. Entries appearing multiple times will be joined in a list under the same
+# key.
+#
+# $ ./mender-inventory-geo
+# country=Poland
+# city=Krakow
+# latitude=50.05750
+# longitude=19.98020
+# ipv4=127.0.0.1
+#
+# The example script collects information on geo localization
+# this can be done in one line:
+# curl https://ipvigilante.com/`curl -4 ifconfig.io 2>/dev/null` 2>/dev/null | jq -r 'select(.status == "success") | "country=" + .data.country_name, "city="+.data.city_name, "latitude="+.data.latitude, "longitude="+.data.longitude, "ipv4="+.data.ipv4 | @text'
+# but in the following we assume busybox version of the utilities.
+
+ATTR_NAME_IP="a-ip"
+ATTR_NAME_CONTINENT="a-continent"
+ATTR_NAME_COUNTRY="a-country"
+ATTR_NAME_CITY="a-city"
```

Comment:
Being an example script, I think that being able to configure the keys of the attributes is adding too much complexity. We should try to get a script as readable as possible.

If you want to keep it, please add a comment explaining what is it, as they are defined here with no comment and used at the end.

Reply:
I do want to keep it, unless you have strong objections. it is cleaner that way.

Reply:
besides: these are the names that will appear in gui and will be used in filters. does putting them at the end of the script make sense to you?

Reply:
Just add a comment as suggested, then.

Reply:
I think hard-coding them in the awk script is what makes most sense, since they are not configureable.

Reply:
I'd at least move them right next to the `awk` invocation.